backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -220,7 +220,6 @@ export class ExperimentAssignmentService {
         target: target,
         user: userId,
```

Comment:
The user here should be the original user instead of just the `userId`. Can you replace this code with
`user: userDoc`